contracts/AxelarGateway.sol

```diff
@@ -264,15 +263,7 @@ contract AxelarGateway is IAxelarGateway, AdminMultisigBase {
         // Prevent setup from being called on a non-proxy (the implementation).
         if (implementation() == address(0)) revert NotProxy();
 
-        (address[] memory adminAddresses, uint256 newAdminThreshold, bytes memory newOperatorsData) = abi.decode(
-            params,
-            (address[], uint256, bytes)
-        );
-
-        // NOTE: Admin epoch is incremented to easily invalidate current admin-related state.
-        uint256 newAdminEpoch = _adminEpoch() + uint256(1);
-        _setAdminEpoch(newAdminEpoch);
-        _setAdmins(newAdminEpoch, adminAddresses, newAdminThreshold);
+        bytes memory newOperatorsData = abi.decode(params, (bytes));
```

Comment:
also should be able to set a new governor in setup

Reply:
Agree, but can do it in a separate PR.

Reply:
nvm, it's an immutable, so need to deploy a new implementation and upgrade to it to update governance/mint limiter